var/spack/repos/builtin/packages/py-pennylane/package.py

```diff
@@ -36,8 +38,8 @@ class PyPennylane(PythonPackage):
     depends_on("py-semantic-version@2.7:", type=("build", "run"))
```

Comment:
Where does the upper bound on scipy come from?

Reply:
It is part of the requirements for v0.31, but not v0.32. Will fix.